### PR TITLE
Feat : Skeletons work 

### PR DESCRIPTION
**Pull request by iamrestrepo**
# Summary
 This PR-- aims to solve issue (#58)--Show skeleton instead of the spinner when loading the differences

I implemented this feature by:

- Adding the react-content-loader library (@^4.2.2)
- Configuring package.json accordingly 
- Implementing a DiffNoData.js file with a component including the skeletons to load when isLoading evaluates to true on the DiffViewer file. 

## Checklist

- [x] I tested this thoroughly
- [ ] I added the documentation in `README.md` (if needed)

### CODE REVIEW

**Comment by iamrestrepo** [comment]
I'd appreciate some feedback regarding the way I implemented the Skeletons. I don't have the "designer eye", only technical knowledge. So I am happy to change anything. 😄

File: src/components/common/NoData/DiffNoData.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React, { Component, Fragment } from 'react'
+import styled from 'styled-components'
+import ContentLoader from 'react-content-loader'
+
+const TitleLoader = () => (
+  <ContentLoader
+    height={40}
+    speed={1}
+    primaryColor={'#ffff'}
```
Comment (lucasbento) [inline src/components/common/NoData/DiffNoData.js]: You can change this to `primaryColor="#FFFFFF"` as you don't need the brackets to pass a string.

File: src/components/common/NoData/DiffNoData.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  line-height: 32px;
+  background-color: #fafbfc;
+  border: 1px solid #e8e8e8;
+  padding: 5px 10px;
+  height: 40px;
+`
+
+const DiffDisplay = styled.div`
+  color: #24292e;
+  background-color: #;
```
Comment (lucasbento) [inline src/components/common/NoData/DiffNoData.js]: Perhaps this can be removed?

File: src/components/common/NoData/DiffNoData.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  border-radius: 3px;
+  padding: 5px 10px;
+  height: 400px;
+  column-count: 2;
+  align-items: center;
+`
+export default class UsefulContentNoData extends Component<Props> {
```
Comment (lucasbento) [inline src/components/common/NoData/DiffNoData.js]: You can remove the `<Props>` type here 😄

File: src/components/common/NoData/DiffNoData.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  render() {
+    return (
+      <Fragment key={'noData'}>
```
Comment (lucasbento) [inline src/components/common/NoData/DiffNoData.js]: The same goes for this prop, you don't need the brackets.

File: src/components/common/NoData/DiffNoData.js (hunk 1 above)
Comment (iamrestrepo) [reply src/components/common/NoData/DiffNoData.js]: Perfect! ✅

File: src/components/common/NoData/DiffNoData.js (hunk 2 above)
Comment (iamrestrepo) [reply src/components/common/NoData/DiffNoData.js]: ✅

File: src/components/common/NoData/DiffNoData.js (hunk 3 above)
Comment (iamrestrepo) [reply src/components/common/NoData/DiffNoData.js]: ✅

File: src/components/common/NoData/DiffNoData.js (hunk 4 above)
Comment (iamrestrepo) [reply src/components/common/NoData/DiffNoData.js]: Got it! ✅ 😃

**Comment by iamrestrepo** [comment]
@lucasbento @pvinis All the suggested changes are on PR #86, let me know if we need something else.

**Comment by pvinis** [comment]
Is this lagginess only happening to me or is it for everyone?
Aug-06-2019 10-08-15

**Comment by pvinis** [comment]
alright. all good then! Leeeeeeeet's do it.

**Comment by RubenSandwich** [comment]
Great job @iamrestrepo!

**Pull request merged**